test/test_wrappers.jl

```diff
@@ -108,4 +108,32 @@ using StatsBase
             end
         end
     end
+    @testset "test adding and removing columns from Tables" begin
+        x, y = rand(10), rand(10)
+        tbl = Table(x=x, y=y)
+        mktempdir(pwd()) do tmp
+            path = joinpath(tmp, "tmp.lh5")
+            lh5open(path, "cw") do lhd
+                lhd["tbl"] = tbl
+                reduce_datastore(lhd, "tbl/y")
```

Comment:
I think it fails on Windows because of this. I wouldn't be surprised if the syntax on Windows is `tbl\\y`.

Reply:
Actually, it was the other way around.. :sweat_smile: